substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/HeapPolicyOptions.java

```diff
@@ -53,6 +53,9 @@
     @Option(help = "How many bytes is enough to allocate an unaligned chunk for an array?  0 implies (AlignedHeapChunkSize / 8).") //
     public static final HostedOptionKey<Long> LargeArrayThreshold = new HostedOptionKey<>(HeapPolicy.LARGE_ARRAY_THRESHOLD_SENTINEL_VALUE);
 
+    @Option(help = "The maximum percentage of heap free after GC to avoid shrinking")
+    public static final RuntimeOptionKey<Integer> MaxHeapFreeRatio = new RuntimeOptionKey<>(70);
```

Comment:
You can avoid the format issue by appending ` //` to the end of the annotation line, like the others do.

Reply:
Or [surround](https://github.com/oracle/graal/blob/6db99e70eb5158923e5041586d876deef1ac2be9/compiler/src/org.graalvm.compiler.core/src/org/graalvm/compiler/core/GraalCompilerOptions.java#L38) the options with `// @formatter:off` and `// @formatter:on`.

Reply:
Thanks! Very good to know! Should be good now.